content/coordination/onboarding/index.en.md

```diff
@@ -60,24 +60,24 @@ For alignment with the R-Ladies mission can you just write an ok as an answer to
 
         -   Country
 
-        -   Organisers
+        -   Organizers
 
         -   Status = Prospective
 
--   Invite organisers to the organisers slack.
+-   Invite organizers to the organizers slack.
 
--   Send the prospective organiser an email with all the most important
+-   Send the prospective organizer an email with all the most important
     information and links (see [Appendix A](#appendix-a))
 
 -   **Other types of requests:**
 
-    -   People may ask to be added to the organiser slack, please check
-        if their name is in the meetup list of co-organisers. If not,
+    -   People may ask to be added to the organizer slack, please check
+        if their name is in the meetup list of co-organizers. If not,
         ask why.
 
-    -   Change list of organisers:\
-        Check and amend the list of organisers here:\
-        Email former organiser for confirmation (see [template B](#appendix-b)).
+    -   Change list of organizers :\
+        Check and amend the list of organizers here:\
+        Email the former organizer for confirmation (see [template B](#appendix-b)).
```

Comment:
"organisers" is consistently used within this document, and is correct UK spelling. I suggest we dont change that.